### ISSUE TITLE
Is www.whirled.com being shut down?

### ISSUE THREAD

**Issue by BAM5** [greyhavens/msoy#31]
Around 12am on 4/8/2017 est the whirled server became unresponsive, which happened occasionally. But I'm somewhat concerned because the cdn, which, being a cdn, should never go down, has also become unresponsive. So I've started to assume the worst.

**Comment by Shadowsych** [comment]
Whirled's public domain has to be repurchased every April 8 of every year. We just need to wait for grey heavens to renew the domain.

**Comment by BAM5** [comment]
I already checked that via the whois. [Registration Expiration Date: 2018-07-17](https://whois.icann.org/en/lookup?name=whirled.com)

**Comment by Shadowsych** [comment]
If it's not the domain then it might be that they need to renew their dedicated server? I used a website to find out what servers they use, and it seems like they use servers hosted by Amazon.

**Comment by BAM5** [comment]
I'm aware of that. Which is why I am concerned since being that it's most likely a cloud application and used a cdn that the only reason it would go down is some sort of catastrophic data center issue, lack of payment, or service cancellation.

**Comment by TurtleShroom** [comment]
Well folks, it looks like we're screwed. It's April 12th, now.

**Comment by iMacG4Projects** [comment]
So, it's broken now. That's what happens to games, and proof that we need Peer-to-peer.

**Comment by Shadowsych** [comment]
You can always check out Synced Online.

www.syncedonline.com
The website used Whirled's source code to make a clone of the game.

**Comment by iMacG4Projects** [comment]
Of course, I didn't know it existed. I plan to make my own clone too, but with Peer-to-Peer to make it (hopefully) last longer. Of course, I'll still check Synced out.

**Comment by Shadowsych** [comment]
If you ever need any developers for hire, then you can always contact me.

I was one of the original founders and developers for Synced Online, and I wouldn't mind helping you out for a good sum of money.

**Comment by Mishuki** [comment]
You could get better developers than the greedy piece of trash Shadow is with volunteers.

Do your research before you hire, please.

**Comment by Madysue** [comment]
So... what happened to Synced?

**Comment by TWSICommunity** [comment]
Synced's still going, just quiet as usual now since what has happened. You are welcome to visit still if you like. But ever since that hack came I had moved it temporary to a private server. And my team didn't like that when I called it Wolves World. I then had to put synced back on because it was a temporary solution.

**Comment by Madysue** [comment]
Oh, I see.

**Comment by Madysue** [comment]
Some hacker guy kinda killed Synced, so what now?

**Comment by lowkeynibba** [comment]
Yea my nibba Cri kinda fucked it but ay no worries xd
